src/pages/image/[uuid].astro

```diff
@@ -27,33 +77,32 @@ const tissueY = (1e6 * image.PhysicalSizeY * image.sizeY).toPrecision(2);
         <section class="vf-content | embl-grid embl-grid--has-centered-content">
             <div></div>
             <div>
-                <h1 class="vf-intro__heading vf-intro__heading--has-tag">{ image.study_accession_id }:{ image.alias }<a href="JavaScript:Void(0);" class="vf-badge vf-badge--primary vf-badge--phases">alpha</a></h1>
-                <h2 class="vf-intro__subheading">{image.name}</h2>
-                {image.original_relpath}
-                <p class="studytitle">{image.study_title}</p>
-            </div>
-            <div>
-                Released {image.release_date}<br>
-                Modified 2023-05-07
+                <h1 class="vf-intro__heading vf-intro__heading--has-tag">{ image_study.accession_id }<a href="JavaScript:Void(0);" class="vf-badge vf-badge--primary vf-badge--phases">alpha</a></h1>
+                <h2 class="vf-intro__subheading">{image_study.title}</h2>
+                <p class="studytitle">{image_dataset.title_id}</p>
+                <a href={ "/bioimage-archive/dataset/" + image_study.accession_id}>
+                    <button class="vf-button vf-button--primary vf-button--sm">Study Information</button>
```

Comment:
I think this should say "Study Page" instead of Study Information, as the latter is already mostly present in the image pages down the page.